### PR TITLE
Switch to Route attribute in controller

### LINKED ISSUE #8: Error with Shopware 6.6.4.0: Make sure there is a loader supporting the "annotation" type.
After installation all I get in the admin UI is the same as when calling `cache:ckear`:

```
root@shopware:/var/www/acme.com# sudo -u www-data bin/console cache:clear

 // Clearing the cache for the prod environment with debug false

In Loader.php line 67:

  Cannot load resource "/var/www/acme.com/vendor/imi/shopware6-category-duplicator/src/Core/Framework/Api/Controller/CloneCategoryController.php".
  Make sure there is a loader supporting the "annotation" type.
```

### PR DESCRIPTION
**Pull request by kdambekalns**
Fixes #8

### CODE REVIEW

**Comment by kdambekalns** [comment]
Tested it locally, worked. I did not check if this still works in versions < 6.6, you might want to adjust the minimum dependency in the composer manifest…

**Comment by amenk** [comment]
@kdambekalns Thanks a lot, I just bumped the Shopware version

File: README.md [hunk 1]
```diff
@@ -12,6 +12,13 @@ bin/console plugin:refresh
 bin/console plugin:install -a iMidiCategoryDuplicator
 ```
 
+## Version Support
+
+* For Shopware 6.4 use the latest 1.x Version
+* For Shopware 6.6 use the latest 3.x Version
+
+We currently do not plan support for Shopware 6.5 - we spare out the 2.x versions, in case anyone wants to add support. 
+
```
Comment (kdambekalns) [inline README.md:21]: Not entirely true… the (previously) used `~6.4` includes all versions < 7.0.0, see [semver.madewithlove.com](https://semver.madewithlove.com/?package=shopware%2Fcore&constraint=~6.4) to believe. 😎

You probably would have wanted to use `~6.4.0` (and for the new release `~6.6.0`) to make it work for the versioning policy of Shopware (where minor versions are in fact major versions)…

Comment (amenk) [reply README.md:21]: true :-)

**Pull request merged**